### PR TITLE
Update jwerl

### PR DESCRIPTION
**Pull request by kamilwaz**
The branch `refactoring/otp-24` was merged in last week so I update it in `rebar.config` and `rebar.lock`.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/146328) / small_tests / 0efb3ac5a49e90d90d7bee553df3479b55f703b5
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3823/146328/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3823/146328/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-10-24_09.40.34/lib.mongooseim.logs/run.2022-10-24_09.40.35/suite.log.html)

File: rebar.config [hunk 1]
```diff
@@ -108,8 +108,7 @@
 
   %%% Other
   {pa, {git, "https://github.com/erszcz/pa.git", {branch, "master"}}},
-  %% MR of jwerl - https://gitlab.com/glejeune/jwerl/-/merge_requests/13
-  {jwerl, {git, "https://gitlab.com/vkatsuba/jwerl.git", {branch, "refactoring/otp-24"}}},
+  {jwerl, {git, "https://gitlab.com/glejeune/jwerl.git", {tag, "1.2.0"}}},
```
Comment (NelsonVides) [inline rebar.config:111]: A hex package was also published, better use that one: https://hex.pm/packages/jwerl

**Review by NelsonVides — CHANGES_REQUESTED** [review]

**Review by pawlooss1 — APPROVED** [review]
Thanks for this fix :)

**Review by NelsonVides — APPROVED** [review]
LGTM 👌🏽

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/146461) / small_tests / b7e66fbf521b913d4a725877877a68eaf98b59e9
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3823/146461/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3823/146461/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-10-24_11.19.24/lib.mongooseim.logs/run.2022-10-24_11.19.25/suite.log.html)

**Pull request merged**